src/components/main_section.tsx

```diff
@@ -0,0 +1,58 @@
+import { ColorMode } from "../pages";
```

Comment:
Please refactor the entire code base to use ENGLISH variable names. It is bad practice, even in a French context, to name variables and file names in languages other than English. Also, please make use of standard naming conventions. A component name is Capitalized, no underscore. 'MainSection.tsx' not 'main_section.tsx'

Reply:
Understood